packages/core/src/storage.ts

```diff
@@ -89,36 +77,37 @@ export const localStorageSetter = (
   clientStorageType: ClientStorageType,
   clientStorage?: ClientStorage
 ): StorageSetter => {
-  if (!clientStorage || clientStorageType === 'localStorage' || clientStorageType === 'web') {
+  if (clientStorageType === 'localStorage' || clientStorageType === 'web') {
     return defaultClientStorageSetter
   }
-
+  if (clientStorageType === 'cookie') {
```

Comment:
as per our discussion, `cookie` type can be checked at the beginning of this function